AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -138,6 +138,8 @@ voici.fr,the-sun.com,thesun.ie,capital.fr,ladbible.com,hitc.com,dualshockers.com
 !
 livoo.fr#$##tarteaucitronAlertBig { display: none!important; }
 livoo.fr#$#html { overflow: auto!important; }
+tomshw.it#$#html, body { overflow: auto !important; }
```

Comment:
1. We do not add more than one selector to a single rule.
2. We already have the rules that change `overflow`.